modules/nf-core/metamaps/mapdirectly/main.nf

```diff
@@ -25,11 +25,12 @@ process METAMAPS_MAPDIRECTLY {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
+    db=`find -L ${database} -name "DB.fa"`
```

Comment:
Maybe also so this in classify?

Reply:
Classify is using the whole path of the database, so I do not think it is needed.

Reply:
I will resolve this.